### PR TITLE
feat: Add AxeMetaData. Modify JSON Serialization so axeViewIds are strings.

### PR DESCRIPTION
**Pull request by chriscm2006**
## Requester Review Items

- [x] Ensure the Pull Request is into develop.
- [x] Test coverage has not gone down.
- [x] Documentation updated or not needed. (Wiki, Issues, etc)
- [x] Angular Type leads to proper Semantic Version.

## Dev Team Review Items

To be filled out by @dequelabs/mobile.

- [x] Ensure the Requester did not lie. Chastise them politely if they did.
- [x] Code Quality review.
- [x] Changes to Axe*.java classes are minimal, appropriate, and versioned properly.
  - [x] Serialized Axe objects have not changed.
  ~~- Public API has not changed.~~

## Code Owner Review Items

To be filled out by @dequelabs/mobileadmin.

- [x] Any Rules package changes lead to proper Semantic Version.
- [x] Security Review.

## Merger Review Items

- [x] Ensure commit message matches title before squashing.

### CODE REVIEW

**Comment by chriscm2006** [comment]
NOTE: This would normally be a breaking change, but we're still in a 0.* release.

File: src/main/java/com/deque/axe/android/AxeView.java [hunk 1]
```diff
@@ -191,7 +191,7 @@ public String speakableTextRecursive() {
     return allAreNull.get() ? null : result.toString();
   }
 
-  public String speakableText() {
+  private String speakableText() {
```
Comment (cyono-deque) [inline src/main/java/com/deque/axe/android/AxeView.java]: Does this change line 24 in `ActiveViewName`?

File: src/main/java/com/deque/axe/android/AxeMetaData.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package com.deque.axe.android;
+
+import com.deque.axe.android.utils.JsonSerializable;
+
+@SuppressWarnings({"FieldCanBeLocal", "unused"})
+class AxeMetaData implements JsonSerializable {
+
+  private final String libVersion = getClass().getPackage().getSpecificationVersion();
+
+  private final String subject;
```
Comment (cyono-deque) [inline src/main/java/com/deque/axe/android/AxeMetaData.java]: `subject` as a field is generic enough to be confusing. Would `applicationIdentifier` work?

File: src/main/java/com/deque/axe/android/AxeView.java [hunk 3]
```diff
@@ -132,7 +132,7 @@ private AxeView(
 
     // This should be the last thing we do in case we decide parent/children relationships
     // contribute to ID calculation.
-    this.axeViewId = this.hashCode();
+    this.axeViewId = Integer.toHexString(this.hashCode());
```
Comment (jldailey13) [inline src/main/java/com/deque/axe/android/AxeView.java]: What does `toHexString` do? Does it convert `axeViewId` (Int) to be in the `0x********....` format?  If so, is this a requirement for iOS to do as well?

File: src/main/java/com/deque/axe/android/AxeView.java (hunk 1 above)
Comment (chriscm2006) [reply src/main/java/com/deque/axe/android/AxeView.java]: This was an auto-ide fix. It should have added an "allow this to be public even though nobody in this repo accesses it" annotation.

File: src/main/java/com/deque/axe/android/AxeMetaData.java (hunk 2 above)
Comment (chriscm2006) [reply src/main/java/com/deque/axe/android/AxeMetaData.java]: I took this field name for AXRL. Though, it looks like they've changed it to testSubject. Fuck AXRL. I'm gonna go with "appIdentifier".

File: src/main/java/com/deque/axe/android/AxeView.java (hunk 3 above)
Comment (chriscm2006) [reply src/main/java/com/deque/axe/android/AxeView.java]: Not a requirement for iOS. A `string` is a `string`. I decided to make it a HEX String for the java library, as the long term plan will be to create a longer string in the future, with lower odds of hash collisions.

**Pull request merged**